test/search.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+// Load external modules
+const Lab = require('lab');
+
+// Load internal modules
+const Search = require('../src/search');
+const CloudSearchStrategy = require('../src/strategies/cloudsearch');
+const StubStrategy = require('../src/strategies/stub');
+
+
+// Test shortcuts
+const lab = exports.lab = Lab.script();
+const expect = Lab.assertions.expect;
+
+let searchParams;
+let search;
+
+lab.describe('#Search class', () => {
+  lab.describe('constructor', () => {
+    lab.beforeEach((done) => {
+      searchParams = {
+        region: 'us-west-1',
+        endpoint: 'https://www.ideapod.com'
+      };
+      done();
+    });
+
+    lab.it('constructs a Search object from this class', (done) => {
+      search = new Search(searchParams);
```

Comment:
This is local to the test case, it shouldn't assign anything to an outside variable. Same below.